app/auth/login.tsx

```diff
@@ -52,12 +73,14 @@ export const Login = ({ errorMessage }: { errorMessage: string }) => {
           ) : null}
           <Flex gap="2" direction="column" align="center">
             <Form action="/auth/github" method="post">
-              <Button type="submit">
-                <Flex gap="1">
-                  <GithubIcon width="16" />
-                  Login with GitHub
-                </Flex>
-              </Button>
+              <WrapperDisabledTooltip isPreview={isPreviewDeployment}>
+                <Button type="submit" disabled={isPreviewDeployment}>
```

Comment:
can we just make a GithubButton component and then if its preview you would wrap conditionally into tooltip? That tooltip wouldn't need isPreview prop and would be semantically just always render children.

Currently WrapperDisabledTooltip is not always rendering the children and its a bit hard to follow this logic

Reply:
Created a login-button component